src/ngraph/frontend/onnx_import/utils/eye.hpp

```diff
@@ -0,0 +1,54 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+
+#include <memory>
+
+#include "common.hpp"
+#include "ngraph/builder/make_constant.hpp"
+#include "ngraph/node.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace eye
+        {
+            /// \brief Creates a square identity matrix.
+            ///
+            /// \param[in] n Order of the resulting matrix.
+            ///
+            /// \return A Constant node representing identity matrix with shape (n, n).
+            template <typename T = double>
```

Comment:
Do we need this to be a template?

Reply:
The template version can be used for other ops, for example `EyeLike` which should support more types.